includes/abstracts/llms-abstract-user-engagement.php

```diff
@@ -28,17 +40,17 @@ abstract class LLMS_Abstract_User_Engagement extends LLMS_Post_Model {
 	public function delete() {
 
 		/**
-		 * Action fired immediately prior to the deletion of a user's earned engagement.
+		 * Action fired immediately prior to the deletion of a user's awarded engagement.
 		 *
-		 * They dynamic portion of this hook, `{$this->model_post_type}`, refers to the engagement type,
-		 * either "achievement" or "certificate".
+		 * They dynamic portion of the hook name, {@see LLMS_Abstract_User_Engagement::$model_post_type `$this->model_post_type`},
+		 * refers to the engagement type, either "achievement" or "certificate".
```

Comment:
I know this is better for your IDE but it's going to look weird when rendered on the codex site with the reference like that...

Reply:
You're right. After testing, I found that it looks good if the curly braces are wrapped in backticks, and the description part is left out. But the link doesn't work, probably because it is to a protected property.

I will remove the `@see` tags from hook DocBlocks.